website/sitePlugin.js

```diff
@@ -0,0 +1,16 @@
+const webpack = require("webpack");
+
+module.exports.default = function sitePlugin() {
```

Comment:
Plugin to fix env variable error when visiting http://localhost:3000/docs/

![image](https://user-images.githubusercontent.com/749374/115874168-14e1b600-a444-11eb-9da9-ce32f8647c80.png)

This should be added to FB internal plugin by @jknoxville

Reply:
Thanks very much for the heads-up and fix @slorber 
I've incorporated this change into the docusaurus plugin, so you can get it by upgrading:

`docusaurus-plugin-internaldocs-fb` to `^0.8.4`